dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogTest.kt

```diff
@@ -62,26 +63,26 @@ import org.mockito.quality.Strictness
 @ForgeConfiguration(Configurator::class)
 internal class DatadogTest {
 
+    lateinit var mockContext: Context
     @Mock
     lateinit var mockLogStrategy: LogStrategy
     @Mock
-    lateinit var mockContext: Context
-    @Mock
     lateinit var mockConnectivityMgr: ConnectivityManager
 
     lateinit var fakeToken: String
 
-    lateinit var packageName: String
+    lateinit var fakePackageName: String
+    lateinit var fakePackageVersion: String
 
     @BeforeEach
     fun `set up`(forge: Forge) {
         fakeToken = forge.anHexadecimalString()
-        packageName = forge.anAlphabeticalString()
+        fakePackageName = forge.anAlphabeticalString()
+        fakePackageVersion = forge.aStringMatching("\\d(\\.\\d){3}")
```

Comment:
Why not using like in the "mockContext" function below the values from the BuildConfig ?

Reply:
Because I want to control that we actually do use the value provided by the context and not a generic value. 

> I don't find it that clear in this PR, so to get it right:
> 
> * `logger.version` - does it come from our SDK version - so, let's say `1.1.0` after we release it?
> * `application.version` - does it come from customer app version - so, let's say `2.6.0` if SDK is used by version `2.6.0` of "Foo" app?

Yes exactly :)

Reply:
Because I want to control that we actually do use the value provided by the context and not a generic value.